csAudioApps/stepSeq
# Reducer

## Description:
- These changes add our project's initial state and a reducer to handle updating state 
## Changes I Made:
- Add initial state(made by @gpanciera ) (and placeholder states) to constants 
- Add reducer with initial functionality and test buttons to MainContainer
- Rename socket connections to be more declarative
- Move socket connection to a file to be imported in components
## How to Test:
- npm install and npm run dev
- initial state should show in react dev tools on load
- uncomment buttons in MainContainer to test reducer functionality
